### ISSUE TITLE
Desktop Application

### ISSUE THREAD

**Issue by OstlerDev** [OstlerDev/PopcornTV#93]
We need a solid Desktop application so that users can more easily install, update, start, and stop PopcornTV without having to mess with the command line at all. This would allow for more users, and less irrelevant bug reports (hopefully).

**Comment by Uokereh** [comment]
Yes a Desktop app for a more easy installation would be wonderful!

**Comment by OstlerDev** [comment]
# Application Preview
![](http://i.imgur.com/UeKjcXs.png)
![](http://i.imgur.com/uAJBbBV.png)
![](http://i.imgur.com/7qy1kOZ.png)
![](http://i.imgur.com/xTf1dM2.png)
![](http://i.imgur.com/6kQDYQV.jpg)
![](http://i.imgur.com/Ws9RwH8.jpg)
![](http://i.imgur.com/u01QgRs.png)

**Comment by JakeRound** [comment]
Awesome work @OstlerDev I'm sure a bunch of people are going are going to find the whole process a lot easier when this is released!

**Comment by Uokereh** [comment]
FANTASTIC WORK!!!

**Comment by OstlerDev** [comment]
If anyone is curious and would like to look thru the code you can view it here: https://github.com/OstlerDev/PopcornTV-App

It is not even close to being done yet and is a work in progress.

**Comment by leoravagnani** [comment]
It would be really cool to have this running on a Mac as a Menu Bar app!

**Comment by Uokereh** [comment]
can't wait fro this decktop app!!! 

its going to be so kick ass!!

**Comment by OstlerDev** [comment]
If anybody would like to test a very alpha build, you can test it here: http://ci.popcorntv.io/job/PopcornTV-Desktop/

**Comment by pattaboy** [comment]
wow! this is awesome. any updates in the near future about the desktop application?
